src/h3lib/lib/algos.c

```diff
@@ -539,12 +539,16 @@ int H3_EXPORT(hexRanges)(H3Index* h3Set, int length, int k, H3Index* out) {
 }
 
 /**
- * Returns the hollow hexagonal ring centered at origin with sides of length k.
+ * Returns the "hollow" ring of hexagons at exactly grid distance k from
+ * the origin hexagon. In particular, k=0 returns just the origin hexagon.
+ *
+ * A nonzero failure return code may happen, for example, if a pentagon is
```

Comment:
Maybe `A non-zero failure code may be returned in some cases, for example if...`